docs/dev/testing.md

```diff
@@ -0,0 +1,38 @@
+# Testing Pull Requests 
```

Comment:
General comment:
I quite often use 
```
pip install git+https://github.com/superdarn/pydarn@<branchname>
```
to install pyDARN straight from the branch in a venv, which might be a good shortcut and doesn't involve too much git?